src/lib/libembind.js

```diff
@@ -17,13 +17,13 @@ var LibraryEmbind = {
   $EmValType__deps: ['_emval_decref', '$Emval', '$readPointer'],
   $EmValType: `{
     name: 'emscripten::val',
-    'fromWireType': (handle) => {
+    fromWireType: (handle) => {
```

Comment:
Maybe convert to method syntax?

Reply:
There is quite a lot more that could be done here, yeah, but knowing how it went in the past, I'm really trying not to get nerd-sniped and not mix too many unrelated changes into the same PR (like you asked :P).

Reply:
fair enough!